src/Data/CatQueue.purs

```diff
@@ -11,15 +11,27 @@ module Data.CatQueue
   , empty
   , null
   , singleton
+  , append
```

Comment:
What's the reasoning behind exporting this?  `append` would be available via `Semigroup` already.